test/denormalizer/ReduxApiStateDenormalizer.spec.js

```diff
@@ -321,6 +321,20 @@ describe('ReduxApiStateDenormalizer', () => {
       assert.isObject(cachedDenormalizedData[STATUS]);
       assert.isObject(cachedDenormalizedData['type2.test'][STATUS]);
     });
+    it('gets object from cache for unexisting item ', () => {
```

Comment:
Actually `false` value is not important here, I have create separate tests for that `test/cache/getReferenceUniqueKey.js`, this is now part of identifiers tests.
Here I am testing if it works when item does not exist in state.